sms_939/models/sms_sponsorship.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    @author: Théo Nikles <theo.nikles@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields
+
+
+class SponsorshipSMS(models.Model):
+    _name = 'sms_sponsorship'
+
+    partner_id = fields.Id(string='partner_id')
+    text = fields.Char(string='text')
```

Comment:
`fields.Text` would allow more characters. String attribute is not needed when the string is the same as the field name. I would also add the `required` attribute, because it doesn't make sense to make a record without text.

Reply:
Done.